includes/sanitizers/class-amp-base-sanitizer.php

```diff
@@ -436,6 +436,11 @@ public function prepare_validation_error( array $error = array(), array $data =
 			if ( ! isset( $error['code'] ) ) {
 				$error['code'] = AMP_Validation_Error_Taxonomy::INVALID_ELEMENT_CODE;
 			}
+
+			if ( ! isset( $error['type'] ) ) {
+				$error['type'] = 'script' === $node->nodeName ? AMP_Validation_Error_Taxonomy::JS_ERROR_TYPE : AMP_Validation_Error_Taxonomy::HTML_ERROR_TYPE;
```

Comment:
This also needs to be a `js_error` if it is an invalidate attribute and the attribute starts with `on`.

Reply:
Good point. https://github.com/Automattic/amp-wp/pull/1373/commits/3e9a81e01947673a754ef6ad39c95cb1023e11f4 adds a `js_error` for `on*` attributes:

<img width="1209" alt="type-js-error" src="https://user-images.githubusercontent.com/4063887/44750894-53c8dd80-aadc-11e8-8819-2b349004143f.png">